ReactSkia/sdk/WindowDelegator.cpp

```diff
@@ -44,7 +44,7 @@ void  WindowDelegator::createNativeWindow() {
                                                  SkSize::Make(windowSize_.width(),windowSize_.height()),
                                                  RnsShell::SubWindow);
   if(window_) {
-    windowContext_ = RnsShell::WCF::createContextForWindow(window_->nativeWindowHandle(),
```

Comment:
As discussed, can we also address issue of using  WindowContext::bufferAge() API when  RNS_SHELL_PARTIAL_UPDATES is disabled